tests/admin/clickhouse_migrations/test_api.py

```diff
@@ -91,3 +92,56 @@ def sort_by_migration_id(migration: Any) -> Any:
     sorted_expected_json = sorted(expected_json, key=sort_by_migration_id)
 
     assert sorted_response == sorted_expected_json
+
+
+@pytest.mark.parametrize("action", ["run", "reverse"])
+def test_run_reverse_migrations(admin_api: FlaskClient, action: str) -> None:
+
+    method = "run_migration" if action == "run" else "reverse_migration"
+
+    with patch(
+        "snuba.settings.ADMIN_ALLOWED_MIGRATION_GROUPS", {"system", "invalid_group"}
+    ):
+        # invalid action
+        response = admin_api.post("/migrations/system/invalid_action/0001_migrations/")
+        assert response.status_code == 404
+
+        # invalid migration group
+        response = admin_api.post(f"/migrations/invalid_group/{action}/0001_migrations")
+        assert response.status_code == 500
+        assert json.loads(response.data) == {"error": "KeyError('invalid_group')"}
```

Comment:
I mentioned this above but I think this is illustrative: the API should return a simple string message, not a stringified Python object. So `{"error": "group not found"}`

Reply:
fixed this